exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,16 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if (mark >= 80)
+    return "A";
+  else if (mark < 80 && mark > 60)
```

Comment:
The `mark < 80` check here is redundant, as any `mark > 80` will be caught in the above conditional, and `return`ed before reaching this line.

You could also use a `switch` statement to clean this up - what might that look like?

Reply:
Thank you, done. 👍